frontend/app/views/spree/checkout/payment/_gateway.html.erb

```diff
@@ -1,9 +1,9 @@
 <%= image_tag 'credit_cards/credit_card.gif', id: 'credit-card-image' %>
 <% param_prefix = "payment_source[#{payment_method.id}]" %>
 
-<p class="field">
+<p class="field" data-hook="card_name">
```

Comment:
I think the general feeling is to remove `data-hook` attributes. To follow this path you could use a class instead, like:

```
<p class="field payment-form-card-name">
```

and add a similar class to the other elements for consistency (number and expiration).

You should be able to hook this element with a class as well, right?

Reply:
Yeah, there shouldn't  be a problem.

I'm going to update the elements to use class.